source/Octopus.Tentacle.Tests.Integration/WorkspaceCleanerTests.cs

```diff
@@ -51,13 +51,7 @@ public async Task WhenScriptServiceIsRunning_ThenWorkspaceIsNotDeleted(TentacleC
             Directory.Exists(startScriptWorkspaceDirectory).Should().BeTrue("Workspace should not have been cleaned up");
 
             File.WriteAllText(waitBeforeCompletingScriptFile, "Write file that makes script continue executing");
-            var runningScriptResult = await runningScriptTask;
-
-            runningScriptResult.LogExecuteScriptOutput(Logger);
-
-            runningScriptResult.ScriptExecutionResult.ExitCode.Should().Be(0, "Script should have completed successfully");
```

Comment:
We _could_ keep the tests to ensure that `runningScriptTask` succeeded (as it was the `Directory.Exists(startScriptWorkspaceDirectory)` check that failed).

I know it's much like `Directory.Exists(startScriptWorkspaceDirectory)`, and technically not part of what's being tested. But still 😁

Reply:
I only recently added those asserts into the test to see if there was something else causing the test to fail!! Log / assert as much as possible to try and track down the issue (which is by design 😆 )

If the script fails or not should not really matter for the intent of the test. We await the execute script operation so the orchestration of the script should have been successful which is all we really care about